README.md

```diff
@@ -112,86 +79,83 @@ Download and install Titanium SDKs
 
 Installs a specific version of the Titanium SDK. If no version is specified, it assumes the latest.
 
-    titanium sdk install
+    ti sdk install
 
-    titanium sdk install <version>
+    ti sdk install <version>
 
-    titanium sdk install <version> --force
+    ti sdk install <version> --force
 
 Download, install <version>, and set as default SDK.
 
-    titanium sdk install <version> --default
+    ti sdk install <version> --default
 
 Download and install the latest version for the specified branch
 
-    titanium sdk install --branch master
+    ti sdk install --branch master
 
 #### sdk uninstall
 
 Uninstalls a Titanium SDK.
 
-    titanium sdk uninstall <version>
+    ti sdk uninstall <version>
 
 #### sdk list
 
 Lists all installed Titanium SDKs. Optionally lists all branches and releases.
 
-    titanium sdk list
+    ti sdk list
 
-    titanium sdk list -b
-    titanium sdk list --branches
+    ti sdk list -b
+    ti sdk list --branches
 
-    titanium sdk list -r
-    titanium sdk list --releases
+    ti sdk list -r
+    ti sdk list --releases
 
-    titanium sdk list -br
-    titanium sdk list --branches --releases
+    ti sdk list -br
+    ti sdk list --branches --releases
 
 ### setup
 
 Reconfigures the Titanium CLI by asking you a series of questions.
 
-    titanium setup
+    ti setup
 
 ### status
 
 Indicates whether you are logged in or not.
 
-    titanium status
+    ti status
```

Comment:
This command hasn't worked in years, please remove all mention of the 'status' command.